backend/src/test/java/de/kathahuth/gymast/service/UserServiceTest.java

```diff
@@ -0,0 +1,81 @@
+package de.kathahuth.gymast.service;
+
+import de.kathahuth.gymast.db.UserMongoDb;
+import de.kathahuth.gymast.model.User;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.*;
+
+class UserServiceTest {
+
+    private final UserMongoDb mockUserMongoDb = mock(UserMongoDb.class);
+    private final UserService userService = new UserService(mockUserMongoDb);
+
+    private User createUserOne() {
+        return User.builder()
+                .username("firstUser")
+                .gymnastname("Firstname1 Lastname1")
+                .build();
+    }
+
+    private User createUserTwo() {
+        return User.builder()
+                .username("secondUser")
+                .gymnastname("Firstname2 Lastname2")
+                .build();
+    }
+
+    @Test
+    @DisplayName("List users should return list of users from db")
+    public void listUsers() {
+        //GIVEN
+        when(mockUserMongoDb.findAll()).thenReturn(List.of(
+                createUserOne(),
+                createUserTwo()));
+        //WHEN
+        List<User> users = userService.listUsers();
+
+        //THEN
+        assertThat(users, containsInAnyOrder(
+                createUserOne(),
+                createUserTwo()));
+    }
+
+    @Test
+    @DisplayName("getUserByUsername should return existing user by username from Db")
```

Comment:
here you mention your method by concatenating the respective words with call case... I was missing that in your previous test description... but it's only for consistency, I guess ^^